### PR TITLE
Fixed parameter types

### PR DESCRIPTION
**Pull request by svycka**

### CODE REVIEW

File: src/AbstractValidator.php [hunk 1]
```diff
@@ -321,8 +321,8 @@ protected function createMessage($messageKey, $value)
     }
 
     /**
-     * @param  string $messageKey
-     * @param  string $value      OPTIONAL
+     * @param  string              $messageKey
+     * @param  string|array|object $value      OPTIONAL
```
Comment (michalbundyra) [inline src/AbstractValidator.php]: Also `null` should be noted here, imo.

Comment (svycka) [reply src/AbstractValidator.php]: then what createMessage() will do as it does not accept null :)

Comment (michalbundyra) [reply src/AbstractValidator.php]: It's fine, see implementation:

```php
if ($value === null) {
    $value = $this->value;
}
```

**Comment by svycka** [comment]
but default value is still null

**Comment by boesing** [comment]
@svycka Thanks for this PR.

To make travis pass, you can either temporarily add `COMPOSER_ROOT_VERSION=2.14.0` to `travis.yml` or `"version": "2.14.0"` to `composer.json` in a single commit.
After this, travis should startup and execute tests.
When anything is green, we can add this to the next release 👍

**Comment by svycka** [comment]
@boesing I am A bit lost do you want something changed? I got an email but I guess you removed the comment.

**Comment by boesing** [comment]
Nah, everything is fine. Thats why I deleted my comment.
Just have to sort some things out regarding this project and merging stuff as we have commits in here which should already be part of the branch but isn't...

**Pull request merged**